.github/workflows/autoremove_labels.yml

```diff
@@ -8,19 +8,22 @@ on:
 
 jobs:
   community_check:
+    if: github.event.action == 'opened'
```

Comment:
This is only needed for the `RemoveNeedsTriageFromMaintainers` job, which previously had this check in its conditional. Moving it here allows us to skip this job entirely if the event type is `pull_request_target`. This also means that if it's skipped, `RemoveNeedsTriageFromMaintainers` is skipped, since it's in the `needs` of that job. As a third benefit, it makes the conditional in that job a bit cleaner looking.